packages/externalModules/package.json

```diff
@@ -8,8 +8,8 @@
   "license": "MIT",
   "private": false,
   "scripts": {
-    "start": "node server.js",
-    "build": "rimraf public && webpack",
+    "start": "http-server -p 3000 --cors",
+    "build": "rimraf public && webpack && cp -R src/configuration public",
```

Comment:
Why not [cp JS](https://www.npmjs.com/package/cp) as an alternative to `cp -R` like `rimraf` for `rm -rf` ?
Because it won't work on non-Unix system

Reply:
I was not aware, thanks